libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -196,13 +201,10 @@ export class LuisRecognizer implements LuisRecognizerTelemetryClient {
         };
         this.includeApiResults = !!includeApiResults;
 
-        // Create client
-        // - We have to cast "creds as any" to avoid a build break relating to different versions
-        //   of autorest being used by our various components.  This is just a build issue and
-        //   shouldn't effect production bots.
-        const creds: msRest.TokenCredentials = new msRest.TokenCredentials(this.application.endpointKey);
-        const baseUri: string = this.application.endpoint || 'https://westus.api.cognitive.microsoft.com';
-        this.luisClient = new LuisClient(creds as any, baseUri);
+        // Create Luis Runtime Client
+        const basePath = 'https://westus.api.cognitive.microsoft.com';
+        this.luisClient = new LuisClient(basePath);
+        this.luisClient.setApiKey(LuisApikeys.apiKeyHeader, this.application.endpointKey)
```

Comment:
This looks like a horrendous bug.... We're not only supporting the `"westus"` region, are we? 😄 

Perhaps some other tests hitting different endpoints are required...

Reply:
We totally missed this, nice catch. We'll take a look into this ASAP.

Reply:
Fixed this to the [original code](https://github.com/microsoft/botbuilder-js/blob/fa7318bf9be24164b990a2afc28b3accce130121/libraries/botbuilder-ai/src/luisRecognizer.ts#L204)